Casks/c/classicftp.rb

```diff
@@ -8,8 +8,8 @@
   homepage "https://www.nchsoftware.com/classic/index.html"
 
   livecheck do
-    url :url
-    strategy :extract_plist
+    url "https://www.nchsoftware.com/classic/versions.html"
+    regex(%r{Version\s+v?(\d+(?:\.\d+)+)\s*</h\d>\s*macOS\s+Release}im)
```

Comment:
```suggestion
    regex(/Version\s+v?(\d+(?:\.\d+)+)[^>]*>\s*macOS/im)
```

I thought this version page looked familiar. We're doing the same thing in `clickcharts` but that regex is a little less explicit, so we may want to use it here as well.

Reply:
Thanks, I'll open a PR to update the regex used in `switch` as well.